vars/spinInfra.groovy

```diff
@@ -91,8 +91,6 @@ def call(Map<String, ?> params) {
 
         sh "terraform --version"
 
-        warnAboutOldTfAzureProvider()
```

Comment:
I don't think this would have worked as intended before, the provider version doesn't show up until you've done a terraform init, but could well show up while you are testing if you've previously done an init...